engine/runtime/src/main/scala/org/enso/compiler/context/ChangesetBuilder.scala

```diff
@@ -58,34 +58,26 @@ final class ChangesetBuilder[A: TextEditor: IndexedSource](
     * @return the computed changeset
     */
   @throws[CompilerError]
-  def build(edits: Seq[TextEdit]): Changeset[A] = {
-
-    val simpleEditOption: Option[TextEdit] = edits match {
```

Comment:
Removing this removes the only way to test the behavior with real IDE that we had so far. Wouldn't it be better to keep this `simpleEditOption` logic in? It hurts nothing, right?